### ISSUE TITLE
Adding items on shortcut makes bug.

### ISSUE THREAD

**Issue by eppc0330** [rathena/rathena#7092]
* **rAthena Hash**: master

* **Client Date**: 20211117

* **Server Mode**: Renewal

* **Description of Issue**: 
  * Result:  Adding items on shortcut makes bug.
    When add usable items on shortcut. After consume all amount, shortcut shows the slot as unknown item, instead of getting blank.
    ![shortcut white potion](https://user-images.githubusercontent.com/47050704/177161420-16b0644e-5384-405e-acb3-4b7de3985648.gif)

  * Expected Result: 
  * How to Reproduce: 
  * Official Information: 
  

* **Modifications that may affect results**:

**Comment by aleos89** [comment]
I want to lean on this being a client issue and not something with rA at all. We don't necessarily define things like this.

**Comment by alisonrag** [comment]
in bRO this is the default behavior
screenshot:
![image](https://user-images.githubusercontent.com/10372732/182228013-a8391367-f32d-4437-8667-3615660582c6.png)

**Comment by Akkarinage** [comment]
I've just double-checked on iRO and kRO and neither of them leave an Unknown Item in the shortcut bar. On both servers I tested with Carrots and Red Herbs.

However, I tested on rA with a 20211103 client and I reproduced the issue.

If bRO did the same as iRO and kRO, then it'd be an rA problem, but if the Official Servers all act differently, I'm inclined to say it's more to do with the individual clients?
